pallets/delegation/src/lib.rs

```diff
@@ -422,23 +459,48 @@ pub mod pallet {
 
 impl<T: Config> Pallet<T> {
 	// Calculate the hash of all values of a delegation transaction
-	fn calculate_hash(
+	fn calculate_delegation_hash_root(
```

Comment:
why the root in the end?

Reply:
Addressed in https://github.com/KILTprotocol/mashnet-node/commit/cbc29eb171e169aa6cc223b017b57d2d7b4303bd.